Cargo.toml

```diff
@@ -11,53 +11,58 @@ version = "0.17.0"
+blake2 = {version = "0.10", default-features = false }
+borsh = { version = "0.10", default-features = false, optional = true }
+bulletproofs = { package = "tari_bulletproofs", git = "https://github.com/tari-project/bulletproofs", version = "4.4.1", default-features = false, optional = true }
+bulletproofs_plus = { package = "tari_bulletproofs_plus", git = "https://github.com/tari-project/bulletproofs-plus", version = "0.3", default-features = false, optional = true }
+curve25519-dalek = { package = "tari-curve25519-dalek", git = "https://github.com/tari-project/curve25519-dalek.git", version = "4.0.3", default-features = false, features = ["alloc", "rand_core", "precomputed-tables"] }
+digest = {version = "0.10", default-features = false }
+getrandom = { version = "0.2", default-features = false, optional = true }
+lazy_static = {git = "https://github.com/rust-lang-nursery/lazy-static.rs.git", rev = "e83d664", default-features = false }
+log = { version = "0.4", default-features = false }
+merlin = { version = "3", default-features = false }
+once_cell = {version = "1.8", default-features = false }
+rand_chacha = { version    = "0.3", default-features = false }
+rand_core = {version = "0.6", default-features = false }
+serde = { version = "1.0", default-features = false, optional = true }
+serde_json = { version = "1.0", default-features = false, optional = true }
 serde-wasm-bindgen = { version = "0.4", optional = true }
-sha3 = "0.9.0"
-thiserror = "1.0.20"
+sha3 = {version = "0.10", default-features = false }
+snafu = {version = "0.7", default-features = false }
 wasm-bindgen = { version = "^0.2", features = ["serde-serialize"], optional = true }
-zeroize = "1"
+zeroize = {version = "1", default-features = false, optional = true }
+rand = { version = "0.8", optional = true }
 
 [dev-dependencies]
-bincode = "1.1.4"
-criterion = "0.3.4"
-rand_chacha = "0.3.1"
-sha2 = "0.9.5"
-wasm-bindgen-test = "0.3.24"
+rand = "0.8"
+bincode = "1.1"
+criterion = {version = "0.5", default-features = false }
+rand_chacha = "0.3"
+sha2 = "0.10"
+wasm-bindgen-test = "0.3"
+zeroize = {version = "1", default-features = false }
+tari_utilities = { git = "https://github.com/tari-project/tari_utilities.git", version = "0.5", default-features = false, features = ["zero", "std"] }
 
 [build-dependencies]
-cbindgen = "0.17.0"
+cbindgen = {version = "0.17", optional = true }
 
 [features]
-default = ["u64_backend"]
-u64_backend = ["curve25519-dalek/u64_backend"]
-simd_backend = ["curve25519-dalek/simd_backend", "bulletproofs/simd_backend"]
-wasm = ["wasm-bindgen", "getrandom/js", "rand/wasm-bindgen", "serde-wasm-bindgen"]
-ffi = []
+default = ["serialize", "bulletproofs", "bulletproofs_plus", "zero", "precomputed_tables", "borsh_ser", "borsh/std"]
+wasm = ["wasm-bindgen", "getrandom/js", "serde-wasm-bindgen", "rand"]
+ffi = ["cbindgen", "rand"]
+zero = ["zeroize", "tari_utilities/zero"]
```

Comment:
I meant to actually raise this in the tari_utilities crate, but this feature would be better named `zeroize`.